src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Distinct.txt

```diff
@@ -0,0 +1,59 @@
+>> Distinct(Blank(), true)
+Blank()
+
+>> Distinct([], true)
+Table()
+
+>> Distinct([{A: 1}, {A: 2}, {A: 3}, {A: 4}], true)
+Table({Result:true})
```

Comment:
Should this return the original source? since formula is always true, meaning everything would evaluate to distinct?